modules/nw-overlapped-sharding.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ingress-cluster-traffic-ingress-controller.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-overlapped-sharding_{context}"]
+== Overlapped sharding example
```

Comment:
just a question now that i think about it: we normally use H1 headers for modules and do the offsetting in the `include` statement. so:
```suggestion
= Overlapped sharding example
```
is there a reason that's different here?

Reply:
again, not a blocking change if needed and can be done in a followup 😄